src/schemas/review.schema.ts

```diff
@@ -22,15 +22,17 @@ export const ReviewSchema = new mongoose.Schema({
   status: { type: String, default: 'PENDING' },
 })
 
-export type Interaction = 'L' | 'D'
+export type InteractionType = 'L' | 'D'
```

Comment:
I name this type as `ReviewInteraction` in [frontend](https://github.com/thinc-org/cugetreg-frontend/blob/29f8f16c67628b43a7fbde32431681e6b2d9ebd3/src/common/types/reviews.ts#L3) 

You can name whatever you want, but pls keep `Review` in the name since it's used only for reviews.